### DISCUSSION TITLE
GitHub for Data Analyst and LowCode Dev

### DISCUSSION THREAD

**Discussion by theoyinbooke** [microsoft/30daysof#184] [Show and tell]
![documentation](https://user-images.githubusercontent.com/13628137/214136042-e125df5b-9781-4472-b0d5-aa3d038fd0ad.png)

This is your moment! You have worked on your projects, you have properly documented them, let's see what you've got.

Go ahead and drop the link to your project repository here. Make sure you have documented your project properly before doing this.

**Comment by Cchioma** [comment]
This is the link to my repository.
https://github.com/Cchioma/weather_report_powerbi.git

**Reply by JusttOdosa** [reply]
This is the link to my repository 🤲https://github.com/JusttOdosa/Pizzalytics

**Reply by Amarabright** [reply]
This is the link to my repository:
https://github.com/Amarabright/Inventory-Management-Analysis

**Reply by theoyinbooke** [reply]
Thank you @Cchioma , @JusttOdosa , @Amarabright  for taking the time to work on your project and for sharing your work.
We will review this, pass feedback, and also find ways to spotlight your project

**Reply by dimeji268** [reply]
This is my Link on the Spending pattern........
https://github.com/dimeji268/Park-n-Ride.git

**Reply by rachelgem** [reply]
https://GitHub.com/RachelGem/run_portfolio.git

**Comment by bellaTHEanalyst** [comment]
This is the link to my repository :
https://github.com/bellaTHEanalyst/powerbi-report-weather-quality-analysis-

**Reply by theoyinbooke** [reply]
Thank you @bellaTHEanalyst  for taking the time to work on your project and for sharing your work.
We will review this, pass feedback, and also find ways to spotlight your project

**Comment by PascalBurume** [comment]
the link to my repository
https://github.com/PascalBurume/United-Nation-Report

**Reply by theoyinbooke** [reply]
Thank you @PascalBurume  for taking the time to work on your project and for sharing your work.
We will review this, pass feedback, and also find ways to spotlight your project

**Comment by Isoken00** [comment]
This is the link to my repository
https://github.com/Isoken00/Air-Data-Quality-In-Beijing

**Reply by theoyinbooke** [reply]
Thank you @Isoken00  for taking the time to work on this project and challenging your power BI skill . 
We will review this, pass feedback, and also find ways to spotlight your project

**Comment by kemiscut** [comment]
This is the link to my repository
https://github.com/kemiscut/TECA-Data-Analysis

**Reply by theoyinbooke** [reply]
Thank you @kemiscut  for taking the time to work on your project and for sharing your work. 
We will review this, pass feedback, and also find ways to spotlight your project

**Comment by Ebuka456** [comment]
This is the link to my repository where I performed social media analytics on a data scraped from Twitter 

https://github.com/Ebuka456/Twitter-Analysis-The-Data-Cleaning-Challenge

**Reply by theoyinbooke** [reply]
Thank you @Ebuka456  for taking the time to work on your project and for sharing your work.
We will review this, pass feedback, and also find ways to spotlight your project

**Comment by tolamoye** [comment]
This is the link to my repository where I cleaned an IMBD dataset using R
https://github.com/tolamoye/IMBD-Data-Cleaning-with-R

**Comment by omotox77** [comment]
This is the link to my repository for the Production Performance using Power BI

https://github.com/omotox77/Project1

**Comment by Amarabright** [comment]
https://github.com/Amarabright/Sales-Performance-Analysis

I carried out another analysis using the Adventure works database

**Comment by Olorlade** [comment]
Here is the link to my repository

https://github.com/Olorlade/Road-Accident-Analysis

**Comment by ArowoloAbimbolaVictoria** [comment]
Here is a link to my repository: 
https://github.com/ArowoloAbimbolaVictoria/Bank-Statement-Analysis

**Comment by iamasprout** [comment]
@theoyinbooke Here's the link to my repository where I used SQL to solve a murder case

https://github.com/iamasprout/Detective-SQL

**Comment by Gbolahant** [comment]
@theoyinbooke you can find the complete project documentation on my GitHub repository: https://github.com/Gbolahant/Inventory-Management-Analysis

**Comment by eoadefila** [comment]
@theoyinbooke here is the link to the documentation of the inventory optimization problem https://github.com/eoadefila/Optimizing-Inventory-Management.

**Comment by Tayo202** [comment]
Here's a link to my most recent repository:
https://github.com/Tayo202/Foodmart 

Your feedback and comments appreciated @theoyinbooke @Amarabright @kemiscut
